src/collection.ts

```diff
@@ -71,6 +71,23 @@ export class Collection {
     return this.pod.collection(this.parentPath);
```

Comment:
The `parent` does not seem to do any checking that the parent directory is actually a collection? Ex: it doesn't check if there is a `_collection.yaml` file?

Reply:
Added a check for if the collection exists before adding to the `parents` array.